conf/livy.conf.template

```diff
@@ -31,6 +31,9 @@
 # If livy should impersonate the requesting users when creating a new session.
 # livy.impersonation.enabled = true
 
+# Logs size livy can cache for each session/batch. 0 means don't cache the logs.
+# livy.spark.logs.size = 200 
```

Comment:
can we name it like `livy.cache-log.size` or something as it applies to both interactive and batch?